Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldManager.cs

```diff
@@ -185,7 +187,50 @@ void _StormshapersPlayer_OnCardRemoved(CardData card, int index)
 
 	}
 
+	void _PointsSystem_CurrencyChanged(int val)
+	{
+		_OverworldUI._ArmouryUI_OnCurrencyChanged(val, _GameStateHolder._ActivePlayer);
+	}
+
+	//Listen to OnShowToggled here because we need to call endTurn and get active player
+	public void _OverworldUI_ArmouryUI_OnShowToggled(bool toggledOn)
```

Comment:
I had it as a listener before but removed the event and forgot to remove this.

If you pause game, you can just call `_ArmouryUI.ToggleShow(false, _GameStateHolder._ActivePlayer);`.